setup.py

```diff
@@ -451,12 +451,7 @@ def write_version(filename: str = str(AIRFLOW_SOURCES_ROOT / "airflow" / "git_ve
     "blinker",
 ]
 
-_devel_only_static_checks = [
-    "pre-commit",
-    "black",
-    "ruff>=0.0.219",
-    "yamllint",
-]
+_devel_only_static_checks = ["pre-commit", "black", "ruff>=0.0.219", "yamllint", "bandit"]
```

Comment:
Those are tools that are installed in the image. You likely need to add bandit as dependency in the .pre-commit-config.yml as "dependencies" (see other examples).